### ISSUE TITLE
crawler changing

### ISSUE THREAD

**Issue by asturi** [nadimtuhin/Facebook-Request-Throttle-WordPress-Plugin#2]
Some requests are shifting over to the meta-externalagent/1.1

**Comment by nadimtuhin** [comment]
Good catch — Facebook has been shifting traffic from `facebookexternalhit` to `meta-externalagent` over the past year.

Both user agents are now detected and throttled in the current release. The plugin also supports a configurable UA list (`$nt_user_agents_to_throttle`) so you can add any future agents without touching core code.

Closing as resolved.
